test/e2e/multiple_policies_for_same_node_test.go

```diff
@@ -0,0 +1,42 @@
+package e2e
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("NodeNetworkState", func() {
+	Context("with multiple policies configured", func() {
+		var (
+			staticIpPolicy = "static-ip-policy"
+			vlanPolicy     = "vlan-policy"
+			ipAddress      = "10.244.0.1"
+			vlanId         = "102"
+		)
+
+		BeforeEach(func() {
+			setDesiredStateWithPolicy(staticIpPolicy, ifaceUpWithStaticIP(*firstSecondaryNic, ipAddress))
+			setDesiredStateWithPolicy(vlanPolicy, ifaceUpWithVlanUp(*firstSecondaryNic, vlanId))
```

Comment:
I would suggest to use secondSecondaryNic so we don't interface with the other policy, if this is what we want.

Reply:
I think the test is better if both policies use the same interface.
Since it seems to work I believe we should keep it.